frontends/p4/tableKeyNames.cpp

```diff
@@ -57,6 +72,9 @@ class KeyNameGenerator : public Inspector {
         name.emplace(expression, l + "[" + r + "]");
     }
 
+    /** Compute a name annotation for @expression.  Eg. `@name("bits")` for
+     * `bits & 0x3` or `0x3 & bits`.
```

Comment:
The name annotation should be inserted by the P4-14 front-end when converting a key mask.
Otherwise we should not handle this specially at all.